.github/workflows/ci.yaml

```
@@ -1,35 +1,48 @@
 name: Continuous Integration
-on: 
- pull_request:
-   types: [opened, synchronize, reopened]
- push:
-   branches:
-     - master
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - master
+  workflow_dispatch:
 
 jobs:
   report:
     name: Report
     runs-on: ubuntu-latest
     steps:
-    - name: ref
-      run: echo ${{ github.ref }} 
-    - name: event_name
-      run: echo ${{ github.event_name }} 
+      - name: ref
+        run: echo ${{ github.ref }}
+      - name: event_name
+        run: echo ${{ github.event_name }}
   build:
     name: Build
+    strategy:
+      matrix:
+        os: [darwin-latest, windows-latest, ubuntu-latest]
```

When trying to understand why the job has been stuck for hours, I realized that, according to https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources, `darwin-latest` does not exist. However, ``macos-latest`` exists as an alias for ``macos-11``

```suggestion
        os: [macos-latest, windows-latest, ubuntu-latest]
```